h2/src/test/org/h2/test/scripts/ddl/createTable.sql

```diff
@@ -131,3 +131,27 @@ SELECT COLUMN_NAME, COLUMN_TYPE FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME
 
 DROP TABLE TEST;
 > ok
+
+CREATE TABLE TEST1(ID BIGINT(20) NOT NULL PRIMARY KEY COMMENT 'COMMENT1', FIELD_NAME VARCHAR(100) NOT NULL COMMENT 'COMMENT2');
```

Comment:
Use `CREATE MEMORY TABLE` here, because by default it can be both `CACHED` or `MEMORY` that is not expected by the following `SCRIPT` test.